Northstar.Client/mod/scripts/vscripts/ui/menu_ns_serverbrowser.nut

```diff
@@ -1097,9 +1097,11 @@ void function ThreadedAuthAndConnectToServer( string password = "" )
 	}
 	else
 	{
+		var reason = NSGetAuthFailReason()
+
 		DialogData dialogData
 		dialogData.header = "#ERROR"
-		dialogData.message = "Authentication Failed"
+		dialogData.message = Localize("#NS_SERVERBROWSER_CONNECTIONFAILED") + "\n\n" + reason
```

Comment:
Is the `reason` parameter already localized here?

Reply:
How about just showing only the reason, since the `Connection failed!` message is kinda redundant given that the title is already `ERROR` and the reason is detailed?

Reply:
> Is the `reason` parameter already localized here?

Nope, and it can't be without major changes elsewhere.

Reply:
Localizing `reason` would make it harder to help others due to language barriers. I say keep it as is

Reply:
It still makes sense to include the error in the localized `NS_SERVERBROWSER_CONNECTIONFAILED` token. Just change it from `Connection Failed or smth` to `Connection Failed or smth\n\n%s1`

Reply:
Other option is to eliminate that part entirely, since it's kinda repetitive anyways.